department-of-veterans-affairs/caseflow
# janet/APPEALS-36751 Set all Caseflow Environments to Operate in UTC

Resolves[ APPEALS-36751](https://jira.devops.va.gov/browse/APPEALS-36751)

# Description
As a Caseflow developer I need all of the environments my code changes will operate in to have consistent timezone configurations. This is especially true for the environments we will be testing implementations in as a precursor to releasing them to production.

## Acceptance Criteria

- [ ] The file at [config/initializers/set_time_zone.rb](https://github.com/department-of-veterans-affairs/caseflow/blob/master/config/initializers/set_time_zone.rb) is removed.

- [ ] A test is devised that can be used by someone with AWS console access to our non-prod VPC to confirm that this change has been applied successfully.

## Testing Plan

1. Log in as a user with a hearing admin role
2. Go to queue
3. Select an appeal to open case details
4. Schedule a **virtual** hearing (convert hearing to virtual if not already)
5. Open AWS console
6. Check Rails log for CreateConferenceJob timezone logging (keywords are "timezones for")
7. Timezone should be UTC, not EST
